front-end/src/app/shared/components/transaction-type-base/transaction-type-base.component.ts

```diff
@@ -141,6 +145,39 @@ export abstract class TransactionTypeBaseComponent implements OnInit, OnDestroy
     this.contactId$.complete();
   }
 
+  // prettier-ignore
+  retrieveMemoText(transactionType: TransactionType, formValues: any) { // eslint-disable-line @typescript-eslint/no-explicit-any
+    const text = formValues['memo_text_description'];
```

Comment:
Just had this thought and maybe we can push it off till later, but if we're using a memo_text_description input, won't it not allow >100 characters even if we're packaging it up into the memo record when we send it to the api?  I'm basically suggesting, should we not just have a dedicated memo_text input and alway remove the memo_text_description input on all forms?

Reply:
Done.  I named it `memo_text_input` as it's a bit easier to not have the formControlName to be the same as the resolved memo_text object